topics/evolution/tutorials/abc_intro_phylo/tutorial.md

```diff
@@ -0,0 +1,1315 @@
+---
+layout: tutorial_hands_on
+title: Phylogenetics - Back to basics
+abbreviations: 
+  ME: Minimum Evolution
+  MP: Maximum Parsimony
+  ML: Maximum Likelihood
+  NJ: Neighbor-Joining
+zenodo_link: 'https://zenodo.org/records/10415242'
+draft: true
+tags:
+  - phylogenetics
+  - evolution
+level: Intermediate
+questions:
+- What information can I get from a phylogenetic tree?
+- How do I estimate a phylogeny?
+- How can I tell whether the phylogeny is reliable?
+- How do phylogenetic models work?
+objectives:
+- Describe the basic concepts behind phylogenetic trees and how they are made
+- Read and interrogate a phylogeny encountered in the literature
+- Use standard tools to align a set of molecular sequences
+- Assess the quality of a molecular sequence alignment and be able to modify it appropriately for phylogenetic analysis
+- Use standard tools to estimate a phylogenetic tree based on a set of aligned molecular sequences
+- Assess the reliability of estimated phylogenies with bootstrapping
+- Explore phylogenetic signal using phylogenetic networks
+time_estimation: 4H
+contributions:
+  authorship:
+    - mcharleston
+  editing:
+    - adamtaranto
+    - burkemlou
```

Comment:
```suggestion
    - burkemlou
  funding:
  - Australian-BioCommons
  - UTas
```

Reply:
Thanks. I'll keep this in mind. I need to check exactly who should be acknowledged as funders for this.